pkg/controller/stats/aggregator.go

```diff
@@ -245,6 +253,32 @@ func (a *Aggregator) ListAntreaClusterNetworkPolicyStats() []statsv1alpha1.Antre
 	return stats
 }
 
+func (a *Aggregator) ListMulticastGroups() []statsv1alpha1.MulticastGroup {
+	stats := make([]statsv1alpha1.MulticastGroup, 0, len(a.groupNodePodsMap))
+	for group, nodePods := range a.groupNodePodsMap {
```

Comment:
there are race conditions between ListMulticastGroups, GetMulticastGroup and doCollect which would crash the program

Reply:
updated with lock added.